src/modules/recruitment/NewProject.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <div class="new__form">
+    <div class="subheading mar__b--1">Create New Project</div>
+    <label for="projectName" class="custom-input__label tagline--bold">Project Name</label>
+    <input
+      id="projectName"
+      type="text"
+      name="projectName"
+      v-model="project_name"
+      placeholder="Please Enter Project Name"
+      class="custom-input mar__b--2"
+    />
+    <label for="projectDuration" class="custom-input__label tagline--bold">Project Duration</label>
+    <input
+      id="projectDuration"
+      type="text"
+      name="projectDuration"
+      v-model="project_duration"
+      placeholder="Please Enter Project Duration"
+      class="custom-input mar__b--2"
+    />
+    <label class="custom-input__label tagline--bold">Research Fields</label>
+    <Multiselect
+        v-model="indexes"
+        mode="tags"
+        :searchable="true"
+        :options="options"
+        :max="4"
+        placeholder="Choose Field..."
+        class="body new-project__multiselect mar__b--2"
+        :createTag="true"
+    />
+    <Button text="Done" v-on:click="addProject" />
+  </div>
+</template>
+
+<script>
+// import {defineComponent, reactive, computed} from 'vue'
```

Comment:
Please remove unwanted imports and comments, to reduce cluter and improve readability